training/amd-bootc/Containerfile

```diff
@@ -3,8 +3,8 @@ FROM ${BASEIMAGE}
 
 ADD rocm.repo /etc/yum.repos.d/rocm.repo
 
-# Include growfs service
-COPY build/usr /usr
```

Comment:
These are not the same thing.

COPY build/usr /usr

Will copy outside of the container from the host into /usr in the container.

cp -r build/usr /usr 

Will copy content from /build/usr to /usr from within the container image.

Reply:
You are completely right Dan. Have to find out a way to make it work from Konflux and undo other PRs with the same change.
Will close it.